xapian-core/tests/api_weight.cc

```diff
@@ -134,6 +134,49 @@ DEFINE_TESTCASE(bm25weight5, backend) {
 }
 
 // Test exception for junk after serialised weight.
+DEFINE_TESTCASE(bm25plusweight3, !backend) {
```

Comment:
There is another test case named `bm25plusweight1` inside `api_nodb.cc` file. Actually, this is the third test case for `BM25PlusWeight` so I thought `bm25plusweight3` would make sense overall.